app/controllers/owners_controller.rb

```diff
@@ -10,19 +10,8 @@ def show
     # Only do this once
     session[:new_supporter] = false if @new_supporter
 
-    @scrapers = @owner.scrapers.includes(:last_run)
-
-    # Split out scrapers into different groups
-    @running_scrapers, @erroring_scrapers, @other_scrapers = [], [], []
-    @scrapers.each do |scraper|
-      if scraper.running?
-        @running_scrapers << scraper
-      elsif scraper.requires_attention?
-        @erroring_scrapers << scraper
-      else
-        @other_scrapers << scraper
-      end
-    end
+    @scrapers = @owner.scrapers.order(:updated_at).includes(:last_run)
```

Comment:
I think you're taking two steps at once. The PR should just address the issue and then additional changes can be made in new PRs, and discussed in new issues. Focused PRs are a million times easier to view cause you've already had most of the discussion on an issue. I think you should revert it back to how it was and then open a new issue about merging the lists if you think it will be better for users.